components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/core/ParAuthServiceImpl.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
Lets add two application-level diagnostic logs to `handleParAuthRequest` and `retrieveParams` methods that will give information on that the par request is being handled and the par request details are being resolved/retrieved.